### PR TITLE
Introduction to Decentralized Storage

### PR DESCRIPTION
**Pull request by oyedeletemitope**

### CODE REVIEW

**Comment by oyedeletemitope** [comment]
@Genesis3800 any update on this

**Comment by Genesis3800** [comment]
Hey @oyedeletemitope ,
I have been busy with a few projects internally, and the team has been busy with Devcon Bogota.
I did go through your draft, and I'll leave you some comments shortly. 
Thank you for your patience.

File: info-articles/introduction-to-decentralized-storage/Article.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+### Bluzelle
+
+Bluzelle is a decentralized database storage network driven by Cosmos and its Byzantine fault-tolerant  (BFT) technology Tendermint, which allows it to be compatible with several blockchains.
+
+Bluzelle provides a decentralized framework for financial data by merging decentralized technologies with edge computing, ensuring that organizations never experience data breaches, network outages, or poor performance, all at a price of a few pennies. Bluzzelle boasts of a lot of features like data immutability, speed, lower cost, high privacy e.t.c
+
+Bluzelle users can make money by renting out their computer storage space. Developers, academics, content providers, and others can rent this space for significantly less than the cost of using centralized cloud services like Amazon's AWS. The cost savings can be used to improve the titles and development processes. In the end, everyone benefits from a more efficient and profitable process.
+
+## Conclusion
```
Comment (Genesis3800) [inline info-articles/introduction-to-decentralized-storage/Article.md:138]: Hey @oyedeletemitope,
I went through your article, and I like that you covered all of the major decentralized storage systems.
I think the list of systems covered is quite exhaustive.

One thing I feel could be added is that decentralized storage systems are not brand new. Think of UTorrent and BitTorrent. Could you add a couple of lines on these?

I will forward your article internally in the meantime, and at the same time, I will be looking forward to your edit.

**Review by Genesis3800 — CHANGES_REQUESTED** [review]

File: info-articles/introduction-to-decentralized-storage/Article.md (hunk 1 above)
Comment (oyedeletemitope) [reply info-articles/introduction-to-decentralized-storage/Article.md:138]: Hello @Genesis3800 . I’ll work on that straight away and get back to you asap.

**Review by Genesis3800 — COMMENTED** [review]
Passing on for peer review internally. Will get back to you ASAP.

**Comment by oyedeletemitope** [comment]
Hello @Genesis3800 any update on the review?

**Comment by Genesis3800** [comment]
Hey @oyedeletemitope ,
We have reviewed your article, but an online plagiarism-checking tool gave us a positive plag report. 
We need to accurately determine the merits of the report before moving ahead with merging your PR. 
I'll get back to you as soon as I can.

**Comment by Genesis3800** [comment]
Hey @oyedeletemitope ,
I would like to sincerely apologize for being so late in getting back to you.
Essentially, it took me some time to arrange for a proper Plag review internally, but as it turns out I have been told that my report gave me a false positive. 
We found around 6% of plagiarism in your submission, which is quite low. 
I would like to move forward on merging your article and processing your payment as soon as possible.

What mode of payment would you prefer?
We offer-
1: Bank payment via SWIFT
2: USDC on the ETHEREUM chain.

**Comment by oyedeletemitope** [comment]
Hello @Genesis3800 thanks for the positive feedback. I would like the bank payment option.

**Comment by Genesis3800** [comment]
> Hello @Genesis3800 thanks for the positive feedback. I would like the bank payment option.

Please drop me your Email here. I will reach out to you over mail.

**Comment by oyedeletemitope** [comment]
Alright.

oyedeletemitope76@gmail.com

**Pull request merged**